_data/toc.yml

```diff
@@ -63,28 +78,29 @@
       children:
         - title: Whitelist command
           url: "floodgate/features/#whitelist-command"
-        - title: Global Api
-          url: "floodgate/features/#what-is-the-global-api"
+        - title: "Commands"
+          url: "floodgate/commands/"
         - title: Skin Uploading
           url: "floodgate/features/#what-is-skin-uploading"
     - title: "Floodgate API"
-      url: "floodgate/api/"
-    - title: "FloodgatePlayer"
-      url: "floodgate/player/"
+      url: "floodgate/floodgate-api/"
+      children:
+        - title: "FloodgatePlayer"
+          url: "floodgate/player/"
+        - title: "Forms and Cumulus"
+          url: "geyser/forms/"
     - title: "Linking"
       url: "floodgate/linking/"
       children:
         - title: Global Linking
           url: "floodgate/linking/#what-is-global-linking"
         - title: Local Linking
           url: "floodgate/linking/#local-linking"
-    - title: "Forms and Cumulus"
-      url: "floodgate/forms/"
-    - title: "Commands"
-      url: "floodgate/commands/"
 - title: Other
   url: "other/"
   links:
+    - title: "Community Geyser Projects"
+      url: "other/community-geyser-projects/"
```

Comment:
Should the community projects come before the official other projects?
I'm fine with it coming before the test server section, but I think that the official other projects should be listed first.